phoenicis-repository/src/main/java/org/phoenicis/repository/repositoryTypes/ClasspathRepository.java

```diff
@@ -195,18 +203,15 @@ public boolean equals(Object o) {
             return false;
         }
 
-        ClasspathRepository that = (ClasspathRepository) o;
-
-        EqualsBuilder builder = new EqualsBuilder();
-
+        final ClasspathRepository that = (ClasspathRepository) o;
```

Comment:
Could we please not mix this kind of changes into this PR? It's not that easy to understand anyways so I would like not to be disturbed by this.